std/experimental/allocator/building_blocks/allocator_list.d

```diff
@@ -173,8 +173,8 @@ struct AllocatorList(Factory, BookkeepingAllocator = GCAllocator)
                 *p = n.next;
                 n.next = root;
                 root = n;
-                return result;
             }
+            return result;
```

Comment:
You are editing an uncovered line, it might be worth to add a unittest to ensure there are no regressions ;-)